### PR TITLE
Add all localized strings for zh

### PR DESCRIPTION
**Pull request by Kagamine**

### CODE REVIEW

**Comment by hishamco** [comment]
:+1:

**Comment by Eilon** [comment]
@kirthik are these strings *supposed* to be missing? I don't see them in any of the RESX files in any language. We only localized the `Hello` string.

**Comment by Kagamine** [comment]
@Elion May I need to put all strings which refered  in the sample by SR[] into resx?

**Comment by Eilon** [comment]
@Kagamine I'm not familiar with the purpose of this sample - I'm just guessing that the string is deliberately left empty.

@DamianEdwards were some of the strings in the sample deliberately left un-localized?

**Comment by Kagamine** [comment]
+1 deliberately left un-localized

**Comment by kirthik** [comment]
@Eilon : Sample was just an example to show localization works. Hence currency, date were being localized according to culture. We only localized `Hello` to show we can retrieve localized resources. We can definitely localize all strings but I am not sure what value it adds.

**Comment by Eilon** [comment]
@kirthik ah ok that's your call, then.

**Comment by Kagamine** [comment]
But it seems that the hello cannot be localized in sample.

**Comment by kirthik** [comment]
@Kagamine : did you try running it in VS or from command line and what culture did you try?

**Comment by Kagamine** [comment]
Yes it is running in VS, I have tried the zh and zh-CN culture.

**Comment by kirthik** [comment]
We don't have support yet for resources running in VS (see https://github.com/aspnet/dnx/issues/2727). You will have to run from command line.

**Comment by Kagamine** [comment]
Oh, thanks! it works.

image

**Comment by kirthik** [comment]
I am ok to merge this PR, if you add localized strings for all `resx` files in the sample. Just adding it in `zh` and `zh-CN` does not make it complete.

**Comment by Kagamine** [comment]
sorry, i am just able to add zh/zh-CN strings, I didn't skilled other languages.

File: samples/LocalizationSample/Startup.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddLocalization(options => options.ResourcesPath = "My/Resources");
         }
 
-        public void Configure(IApplicationBuilder app, IStringLocalizer<Startup> SR)
+        public void Configure(IApplicationBuilder app)
```
Comment (hishamco) [inline samples/LocalizationSample/Startup.cs]: Why you remove ```IStringLocalizer``` ?!!

**Comment by hishamco** [comment]
@Kagamine please add the values on the resx only and squash the commits into a single commit
thanks

**Comment by Kagamine** [comment]
ok

**Comment by hishamco** [comment]
FYI you can squash the commits using git command line instead of creating a new PR

**Comment by Kagamine** [comment]
QAQ, I didn't skilled that.

**Comment by hishamco** [comment]
Have a look to this http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/

**Comment by Kagamine** [comment]
thanks

**Pull request closed without merging**